aviary/utils/process_input_decks.py

```diff
@@ -59,18 +59,29 @@ def create_vehicle(vehicle_deck='', verbosity=Verbosity.BRIEF):
     aircraft_values.set_val('mass_defect', val=10000, units='lbm')
     aircraft_values.set_val(Settings.PROBLEM_TYPE, val=ProblemType.SIZING)
     aircraft_values.set_val(Aircraft.Electrical.HAS_HYBRID_SYSTEM, val=False)
-
+    
+    initial_guesses = {
+        # initial_guesses is a dictionary that contains values used to initialize the trajectory
+        'actual_takeoff_mass': 0,
+        'rotation_mass': .99,
+        'operating_empty_mass': 0,
+        'fuel_burn_per_passenger_mile': 0.1,
```

Comment:
something I've been meaning to change, but haven't gotten around to, is changing the default values for rotation_mass and fuel_burn_per_passenger_mile to 0, and then adding something like:
```
if rotation_mass == 0:
	rotation_mass = .99
```
to be consistent with the rest of the variables.
Would you be ok with adding that in here as well?

Reply:
I've done this within the same 'create_vehicle' method. I wondered about putting it in 'initial_guessing', but this isn't always called... other option would be to put it at the end of the create_vehicle method, but then if a user has specified a zero, we'd be overwriting that with a default, which I think would be undesirable behavior... (I'm not sure what happens if the initial guess for rotation_mass or fuel_burn_per_passenger is zero, but i'd want it to error if I manually sent it a bad value, rather than overwriting my input with a default)

As an aside it looks like this variable should actually be the absolute mass of the aircraft on rotation (in lbs):
![image](https://github.com/OpenMDAO/Aviary/assets/165826178/948ad7d2-8dae-44d6-b7a2-4831d2689818)

It seems we're setting the default to 0.99lbs, which seems unnecessary - could we just use 0?

Reply:
sorry, I meant to say that this should be initial_guessing with the rest of them.

In initial_guessing, this will be converted to a mass in pounds
![image](https://github.com/OpenMDAO/Aviary/assets/51181861/9794898b-13b6-4597-8235-3aeda2b9ab4f)

I'm envisioning that rotation_mass would be handled similar to cruise_mass_final
![image](https://github.com/OpenMDAO/Aviary/assets/51181861/6552bbe7-e507-43ff-8374-1a17ff5275c8)

Reply:
ok gotcha - I've updated this now. initial_guessing is only called:
![image](https://github.com/OpenMDAO/Aviary/assets/165826178/c0adac3b-dcaa-4e6a-bde0-80e928450197)
So if those conditions aren't met we'll be defaulting the initial_guesses to 0 (which is a change in behavior to what is currently in the code)

Reply:
As far as I'm aware, the initial guesses aren't used by the height energy missions or the FLOPS mass calculations, so this shouldn't cause any problems